web/components/input/buttons/CheckboxGroup/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InputContext = React.createContext();
+
+const Checkbox = ({ name, value, label, checked, disabled, onChange }) => (
+  <InputContext.Consumer>
+    {context => (
+      <label className="checkbox">
+        <input
+          type="checkbox"
+          name={name || context.name}
+          value={value}
+          defaultChecked={checked || context.defaultChecked}
+          disabled={disabled || context.disabled}
+          onChange={onChange || context.onChange}
+        />
+        <span className="checkmark"></span>
+        <span>{label}</span>
+      </label>
+    )}
+  </InputContext.Consumer>
+);
+
+Checkbox.propTypes = {
+  // name should be the same amongst checkboxes in a checkbox group
+  name: PropTypes.string,
+
+  // input value for checkbox
+  value: PropTypes.string.isRequired,
```

Comment:
wouldn't this be a boolean?

Reply:
Which part are you referring to?

Reply:
checkbox value is either true or false, right? unless value is the label, but it looks like that's covered by `name || context.name`